examples/amr_human_paired_sample_with_rpm.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Do your dag changes cause the previous amr example jsons to break? If so, please either update or remove them.